### PR TITLE
fix Issue #46

### PR DESCRIPTION
**Pull request by jnstahl**
comment out verbose printf statements which were causing compilation issues on windows

### CODE REVIEW

File: windows/flutter_nsd_plugin.cpp [hunk 1]
```diff
@@ -385,7 +385,7 @@ void call_HandlePTRRecord(const void* p, const void* base, boolean last, STRING_
   }
 
   void call_HandleTXTRecord(const void* p, const void* base, boolean last, STRING_ARG_DECL(key), STRING_ARG_DECL(value)) {
-    ((MdnsRequest*)p)->callbackTXT(base, last, STRING_ARG_CALL(key), STRING_ARG_CALL(value));
+    ((MdnsRequest*)p)->callbackTXT(base, last, STRING_ARG_CALL(key), STRING_ARG_CALL(key));
```
Comment (Nimrodda) [inline windows/flutter_nsd_plugin.cpp:388]: why value is changed to key here?

**Comment by jnstahl** [comment]
Seems like a merge issue on my side? Weird I didn't see that

File: windows/flutter_nsd_plugin.cpp (hunk 1 above)
Comment (jnstahl) [reply windows/flutter_nsd_plugin.cpp:388]: Huh. I fixed this earlier. Why did this regress?

Comment (Nimrodda) [reply windows/flutter_nsd_plugin.cpp:388]: I have no idea :D but I guess it should be `value`, right?

**Comment by Nimrodda** [comment]
> Seems like a merge issue on my side? Weird I didn't see that

yeah, no worries. whenever you have the time.

**Comment by Nimrodda** [comment]
I'll close this one then

**Pull request closed without merging**